modules/Link.js

```diff
@@ -20,6 +20,21 @@ function isEmptyObject(object) {
   return true
 }
 
+function isLinkActive(router, props) {
```

Comment:
Note my caveat in https://github.com/reactjs/react-router/issues/470#issuecomment-217016892 – if you hit this from clicking on a link, then stuff might well get batched together, but it won't happen if the transition is triggered from something other than a React event handler.